raijin_scripts/execute_fractional_cover/run

```diff
@@ -95,7 +92,13 @@ echo "**********************************************************************"
 echo ""
 echo "Executing fractional cover for ${YEAR} ${PRODUCT} with tag ${TAG}"
 ##################################################################################################
-# Run dea-submit-fractional cover process
+# Run datacube-fc submit two stage PBS job
 ##################################################################################################
 datacube-fc submit -v -v --project "${PROJECT}" --queue "${QUEUE}" --year "${YEAR}" --app-config "${APP_CONFIG}" --tag "${TAG}"
 EOF
+
+sleep 60s # Wait 60s before moving the submission log
+# Move submission log file to fc work directory
+searchstring='Created work directory '
+dirpath=$(sed "s:.*$searchstring/::" <<< "$(grep "$searchstring" "$SUBMISSION_LOG")")
+if [[ -d /"$dirpath" && -f "$SUBMISSION_LOG" ]]; then cp -p -r "$SUBMISSION_LOG" /"$dirpath/$LOG_FILENAME" && rm -rf "$SUBMISSION_LOG"; fi
```

Comment:
Why `cp` then `rm` instead of just `mv`?

Also, the `-r` options are also confusing since the log file is a single file, and it makes it seem like it could be a directory.

Also, `$dirpath` can quite easily be "", which means we'll check that `/` is a dir, and then try to copy our log there, which will fail. It's not a particularly nice failure mode.